docs/core/compatibility/networking/7.0/socket-end-closed-sockets.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Breaking change: Socket.End methods don't throw ObjectDisposedException"
+description: Learn about the .NET 7 breaking change in networking where Socket.End methods no longer throw ObjectDisposedException when the socket is closed.
+ms.date: 09/14/2022
+---
+# Socket.End methods don't throw ObjectDisposedException
+
+`System.Net.Sockets.Socket.End*` methods (for example, <xref:System.Net.Sockets.Socket.EndSend%2A>) throw a <xref:System.Net.Sockets.SocketException> instead of an <xref:System.ObjectDisposedException> if the socket is closed.
+
+## Previous behavior
+
+Previously, the [affected methods](#affected-apis) threw an <xref:System.ObjectDisposedException> for closed sockets.
+
+## New behavior
+
+Starting in .NET 7, the [affected methods](#affected-apis) throw a <xref:System.Net.Sockets.SocketException> with <xref:System.Net.Sockets.SocketException.SocketErrorCode> set to <xref:System.Net.Sockets.SocketError.OperationAborted?displayProperty=nameWithType> for closed sockets.
+
+## Version introduced
+
+.NET 7 Preview 7
+
+## Type of breaking change
+
+This change can affect [binary compatibility](../../categories.md#binary-compatibility).
+
+## Reason for change
+
+The [asynchronous programming model (APM)](../../../../standard/asynchronous-programming-patterns/asynchronous-programming-model-apm.md) APIs are those named `Begin*` and `End*`. Starting with .NET 6, these legacy APIs are backed with a `Task`-based implementation as part of an effort to consolidate and simplify the `Socket` codebase. Unfortunately, the 6.0 implementation leaked unobserved `SocketException` exceptions. This happened even when the APIS were used correctly, meaning that the calling code always invoked the `End*` methods, including when the socket was closed).
```

Comment:
@gewarren excuse me, I was OOF, so I'm a bit late to the game here.

"Leaks unobserved exceptions" is an ad-hoc term I used, and it might need clarification for some of the readers. The exact meaning is that there will be unexpected events raised on [TaskScheduler.UnobservedTaskException](https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskscheduler.unobservedtaskexception?view=net-6.0), this is why I linked `TaskScheduler.UnobservedTaskException` to the word "unobserved" in my original text.

This comment is sort-of nitpicking, up to you if you address it or not.